weldx/transformations/cs_manager.py

```diff
@@ -787,7 +780,8 @@ def create_cs_from_euler(
         angles,
         degrees: bool = False,
         coordinates: types_coordinates = None,
-        time: pd.DatetimeIndex = None,
+        time: Union[types_time_like, Time, None] = None,
+        time_ref: Union[types_timestamp_like, None] = None,
```

Comment:
Since we want to get rid of the `time_ref` parameter in the long run anyway I would prefer not to introduce this for existing functions, the user should supply a proper `Time` object here

Reply:
I am not 100% sure if we should really get rid of the `time_ref` since it gives the user the flexibility to pass in whatever he wants without any need to involve another class. The inputs are forwarded to `Time` anyways. I like this kind of flexibility.

Apart from that, I think we should remove or merge most of the LCS construction methods (`from_xyz` etc.) and the corresponding `create` methods on the CSM.

Reply:
> I am not 100% sure if we should really get rid of the `time_ref` since it gives the user the flexibility to pass in whatever he wants without any need to involve another class. The inputs are forwarded to `Time` anyways. I like this kind of flexibility.

imo it bloats the parameters and docstrings of many functions while providing the exact functionality of the `Time` class, maybe keep it as short as possible and add some more flexibility later where we really need it?
We could also support tuple arguments for `time` where the second item is the reference time so no explicit `Time()` call needed (we should add that to the `Time` constructor)

> 
> Apart from that, I think we should remove or merge most of the LCS construction methods (`from_xyz` etc.) and the corresponding `create` methods on the CSM.

I am also for removing them

Reply:
I think that this is a discussion for the future and not tied to this PR. The `create_` methods were just thin wrappers around the LCS `from_` functions. The modified places missed the `time_ref` parameter that is part of the corresponding `from_` function. I added it because I had to fix the type hints of all those functions. I do not see the point in inverting the modifications when the `create_` functions still possess this parameter.